dashboard/src/utils/constructDate.js

```diff
@@ -1,3 +1,3 @@
-export const constructNewDate = (date) => {
-  return new Date(`${date.split(":")[0]}T00:00:00`);
+export const constructUTCDate = (date) => {
+  return date.setMinutes(date.getMinutes()+date.getTimezoneOffset());
 };
```

Comment:
Can we fix this in a follow-up PR?

Reply:
Obviously we could, but I think that this code and the code surrounding it is foundational, and therefore it's worth spending the time to get it right.

There's very little advantage to converting the string to a `Date` before calling this function rather than doing it inside this function; whereas, doing inside the function encapsulates the conversion from local time to UTC, so that we don't leak local times.  Committing the code without this change leaves us exposed to confusion about what `Date` objects represent, so it's best to address this before merging.